src/Validation/Validation.php

```diff
@@ -1424,6 +1424,17 @@ public static function isArray($value)
     }
 
     /**
+     * Check that the input value is a scalar.
+     *
+     * @param mixed $value The value to check
+     * @return bool
+     */
+    public static function isScalar($value)
+    {
+        return is_scalar($value);
```

Comment:
We don't have `isIntegerOrNull()`, and `isInteger()` doesn't pass null values. So I don't think we need to have `isScalarOrNull()`.